src/modules/Classroom/containers/ClassroomView.tsx

```diff
@@ -1,14 +1,125 @@
+import { Accordion, Button } from 'flowbite-react';
 import React from 'react';
+import {
+  HiOutlineAnnotation,
+  HiOutlineBookOpen,
+  HiOutlinePlay,
+} from 'react-icons/hi';
 import { useParams } from 'react-router-dom';
 
+import CardReminder, { CardReminderProps } from '../components/CardReminder';
+
 type RouteParams = {
   classroom: string;
 };
 
 const ClassroomView: React.FC = () => {
   const params = useParams() as RouteParams;
 
-  return <>{JSON.stringify(params)}</>;
+  const cardReminders: CardReminderProps[] = [
+    {
+      title: 'Exames finais',
+      data: '18/02/2022',
```

Comment:
Please change it to something like
````
{
  title: '',
  reminderAt: ''
}
````

to make it easy to extract to the mock server later.